transcom/mymove
# MAIN B-21549

[INT PR](https://github.com/transcom/mymove/pull/13931)

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21549)

## Summary

Fixing some silly little spacing errors when shipments are listed out.

Out with the old, in with the new.
![Screenshot 2024-10-16 at 10 01 16 AM](https://github.com/user-attachments/assets/44c1cdb0-24ad-41f0-9163-72ee6c397918)
![Screenshot 2024-10-16 at 10 28 50 AM](https://github.com/user-attachments/assets/0edcf4dc-5a6f-4ea8-8407-52cfd6685e89)
![Screenshot 2024-10-16 at 10 29 13 AM](https://github.com/user-attachments/assets/6c683595-8fa0-498c-8ce6-addab6b63e65)

With UB
![Screenshot 2024-10-16 at 12 48 15 PM](https://github.com/user-attachments/assets/b9840d60-4429-4661-bc18-e7cdf1efdfbb)


### How to test

1. Access MM as a customer
2. Create a shipment of each kind
3. Verify that the spacing is not all ugly and wonky with the shipment IDs
